app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApiController
+      skip_before_action :authenticate_user!, only: :create
+
+      def create
+        new_bs = new_book_suggestion
+
+        if new_bs.save
+          render json: new_bs, serializer: BookSuggestionSerializer, status: :created
+        else
+          render json: { errors: new_bs.errors }, status: :unprocessable_entity
+        end
+      end
+
+      private
+
+      def create_bs_params
+        params
+          .require(:book_suggestion)
+          .permit(:synopsis, :price, :author, :title, :link, :publisher, :year)
+      end
+
+      def new_book_suggestion
+        if current_user.present?
```

Comment:
Are you sure you can do this? I don't think you can call `current_user` if you are skiping `authenticate_user!`

Reply:
@facucompe yes, no problem. It still returns the logged in user or nil (when there is no logged in user).